src/main/resources/jira-template.ftl

```diff
@@ -1,16 +1,25 @@
-<#assign rowTotalDuration = 0?number>
 <#if getDescription()?has_content>${getDescription()}</#if>
+
 <#if getNarrativeType() == "NARRATIVE_AND_TIME_ROW_ACTIVITY_DESCRIPTIONS">
  <#list getTimeRows() as timeRow>
-  <#assign rowTotalDuration += timeRow.getDurationSecs()>
   |${timeRow.getActivity()}|${timeRow.getDescription()!"N/A"}|
  </#list>
 </#if>
-<#if getDescription()?has_content || getNarrativeType() == "NARRATIVE_AND_TIME_ROW_ACTIVITY_DESCRIPTIONS">
- ${'\n'}
-Total worked time: ${rowTotalDuration?string.@duration}
+
+Total worked time: ${getTotalDuration(getTimeRows())?string.@duration}
```

Comment:
Remove worked time and chargeable time lines. This is what we want:

![rendered](https://user-images.githubusercontent.com/305692/52926215-d0dc5d80-336f-11e9-9333-c388e824269d.png)

Reply:
Can you check whether experience weighting as already been applied to the total duration logged into Jira? If so, maybe change "Experience factor" to say "Applied experience weighting".

Reply:
Yes, it is already applied. Will apply suggestions